app/admin/RTMediaAdmin.php

```diff
@@ -358,15 +357,15 @@ public function rtmedia_premium_addon_notice() {
 					</p>
 				</div>
 				<script type="text/javascript">
```

Comment:
We can move these JS codes in to a separate/existing JS file to make the php code clean.